diesel_cli/src/database.rs

```diff
@@ -303,19 +320,22 @@ mod tests {
 
     #[test]
     fn split_pg_connection_string_returns_postgres_url_and_database() {
```

Comment:
New unit test added. This function should every valid Postgres connection string. On the other side of the fence, not all valid MySQL connection strings are supported. I could not find official documentation of MySQL connection string grammar (found the JDBC and ODBC, but not the c library), so I just used the grammar documented in **diesel** itself (http://docs.diesel.rs/diesel/mysql/struct.MysqlConnection.html).